src/styles/StyleUtils.ts

```diff
@@ -275,28 +281,28 @@ function getZoomSizingStyle(
     containerHeight: number,
     containerWidth: number,
     isLoading: boolean,
-): ViewStyle | CSSProperties | undefined {
+): ViewStyle | undefined {
     // Hide image until finished loading to prevent showing preview with wrong dimensions
     if (isLoading || imgWidth === 0 || imgHeight === 0) {
         return undefined;
     }
-    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px`;
-    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px`;
+    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px` as ViewStyle['top'];
+    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px` as ViewStyle['left'];
 
     // Return different size and offset style based on zoomScale and isZoom.
     if (isZoomed) {
         // When both width and height are smaller than container(modal) size, set the height by multiplying zoomScale if it is zoomed in.
         if (zoomScale >= 1) {
             return {
-                height: `${imgHeight * zoomScale}px`,
-                width: `${imgWidth * zoomScale}px`,
+                height: `${imgHeight * zoomScale}px` as FlexStyle['height'],
+                width: `${imgWidth * zoomScale}px` as FlexStyle['width'],
```

Comment:
The same recommendation for other cases.
```suggestion
                height: `${imgHeight * zoomScale}px` as DimensionValue,
                width: `${imgWidth * zoomScale}px` as DimensionValue,
```